docker/linux/android/Dockerfile

```diff
@@ -52,6 +52,7 @@ ARG CMDLINE_URL=https://dl.google.com/android/repository/${TOOLS}
 ENV ANDROID_SDK_ROOT=/root/starboard-toolchains/AndroidSdk/
 
 RUN cd /tmp \
+  && rm -rf ${ANDROID_SDK_ROOT} \
```

Comment:
This change can be independently submitted, right?
Why is it part of the m114 upgrade?

Reply:
This fixes a failing test , idk why it's failing on this branch although it's not needed.
If you really want it , i can add it in a separate branch and submit it to trunk and it'll go away as part of the merge conflict resolution.  Brian has resolved merge conflicts once and so have I in this feature branch repo